docs/external-node.md

```diff
@@ -0,0 +1,577 @@
+# ExternalNode
+
+## Table of Contents
+
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Name and Namespace](#name-and-namespace)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+  - [Antrea NetworkPolicy configuration](#antrea-networkpolicy-configuration)
+  - [Bypass Antrea NetworkPolicy](#bypass-antrea-networkpolicy)
+- [RBAC for antrea-agent](#rbac-for-antrea-agent)
+- [VM network configuration](#vm-network-configuration)
+- [Install Antrea Agent on VM](#install-antrea-agent-on-vm)
```

Comment:
Probably put "VM network configuration" before "RBAC for antrea-agent", as it may be more natural to talk about VM configuration changes after installation.

Reply:
If "VM network configuration" and "RBAC for antrea-agent" are about installation, do you think I should put these two under section "Install Antrea Agnet on VM" ? @jianjuns

Reply:
According to the offline discussion, I would leave the two sections at the same level as "VM network configuration"